internal/service/advancedcluster/resource_advanced_cluster_test.go

```diff
@@ -185,9 +185,6 @@ func TestAccClusterAdvancedCluster_unpausedToPaused(t *testing.T) {
 }
 
 func TestAccClusterAdvancedCluster_pausedToUnpaused(t *testing.T) {
-	// TODO: Already prepared for TPF but getting this error:
-	// POST: HTTP 400 Bad Request (Error code: "CANNOT_CREATE_PAUSED_CLUSTER"). Detail: Cluster test-acc-tf-c-5570724607982544726 cannot be created in a paused state.
-	acc.SkipIfAdvancedClusterV2Schema(t)
 	var (
```

Comment:
the prod code is already changed?

Reply:
Yes, this was fixed in a commit to #2855